reconcile/cli.py

```diff
@@ -376,6 +378,8 @@ def include_trigger_trace(function):
 
 
 def run_integration(func_container, ctx, *args, **kwargs):
+    faulthandler.enable(all_threads=True)
+    faulthandler.register(SIGUSR1)
```

Comment:
is this meant for manual troubleshooting by sending the event within the running pod?
or do we intend to have some tooling doing that somehow so that the desired output shows up in the pod logs?

Reply:
This is for manual troubleshooting when integrations are stuck. We can certainly build tooling around it later.

Reply:
manual is for sure sufficient for now. thanks for adding this functionality 👍